python/paddle/distribution/kl.py

```diff
@@ -68,9 +65,15 @@ def kl_divergence(p, q):
 def register_kl(cls_p, cls_q):
     """Decorator for register a KL divergence implemention function.
 
+    The ``kl_divergence(p, q)`` function will search concrete implemention 
+    function registered by ``register_kl``, according to multi-dispatch pattern. 
+    If find the implemention function, it will return the result, or not will 
+    raise ``NotImplementError`` exception. User can register implemention 
+    funciton by the decorator. 
```

Comment:
1. implemention functions (plural forum);
2. If an implemention function is found;
3. ortherwise, it will raise a ``NotImplementError`` exception;
4. Users
5. functions